Documentation/RenderingDocs/Index.rst

```diff
@@ -68,6 +68,89 @@ Make sure that `Docker <https://www.docker.com/>`__ is installed on your system.
             docker run --rm --pull always -v ${PWD}:/project -it ghcr.io/typo3-documentation/render-guides:latest --config=Documentation
             start "Documentation-GENERATED-temp/Index.html"
 
+Rendering with more WYSIWYG-feeling (automatic re-rendering)
+------------------------------------------------------------
+
+Often, especially in the later stages of creating documentation, you
```

Comment:
I would start with a sentence about what it does. Otherwise you might loose readers in the first paragraph

Reply:
I added such a sentence, but it's not really my style ;) I always first want to establish the groundrules so for me I'd first want to postulate the problem/setup and then work from there on how to improve it.

But maybe this works now?

Reply:
Better. We are writing for the internet here and ppl are volatile. Studys show they decide within the first few seconds if they keep reading or not.